config/content/Sponsors.js

```diff
@@ -137,6 +145,13 @@ export default {
       link: 'https://www.voiceflow.com/',
       alt: 'Voiceflow',
     },
+    {
+      src: 'https://res.cloudinary.com/dscnitrourkela/image/upload/project-oregano/sponsors/delta/jzu4m46qrkuheumwzrcx.png',
```

Comment:
This logo looks so ugly 🤣 try once without background!!

Reply:
@ORKO06 do update the cloudinary link.